test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -23,9 +23,58 @@
 using namespace std;
 using namespace yul::test::yul_fuzzer;
 
+const std::string yul::test::yul_fuzzer::createHex(std::string const& _hexBytes)
+{
+	std::string tmp = std::string(_hexBytes);
```

Comment:
```suggestion
	string tmp = string(_hexBytes);
```

Reply:
```suggestion
	string tmp{_hexBytes};
```

Reply:
`std::string` is a funny story.

The fuzzing toolchain depends on `libc++` (as against libstdc++) and for some reason (although we are doing `using namespace std;`), the linker (clang++) links `string` to a libc++ implementation of it which is slightly different.

This leads to linking errors when we mix (std::string and string implementation in libc++). That's the reason I explicitly reference `std::string`. Other comments are valid though.